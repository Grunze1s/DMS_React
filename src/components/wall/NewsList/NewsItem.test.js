import React from 'react';
import NewsItem from './NewsItem';
import {
    configure,
    shallow
} from 'enzyme';
import EnzymeAdapter from 'enzyme-adapter-react-16';
// JestHook.mock('expo-font');
configure({
    adapter: new EnzymeAdapter
})


const setup = (props = {}, state = null) => {
    return (shallow( < NewsItem {...props}  />)
    )
}
const dummy_data={
   data:{
       image:"image.png",
       descriptio:"abcdef"
   }
}
describe('<NewsItem />', () => {
  

it("NewsItem render", () => { 
        const wrapper= setup({...dummy_data});
        expect(wrapper).toMatchSnapshot();
  
})
})

