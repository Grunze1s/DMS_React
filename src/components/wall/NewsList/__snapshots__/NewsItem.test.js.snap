// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NewsItem /> NewsItem render 1`] = `ShallowWrapper {}`;
